### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: docs/04_tying_in_the_market_maker.md [hunk 1]
```diff
@@ -47,6 +48,7 @@ event LMSRMarketMakerCreation(
 );
 ```
 
+Rest of `migrations/07_create_lmsr_mm.js`:
```
Comment (germartinez) [inline docs/04_tying_in_the_market_maker.md]: `/migrations/07_create_lmsr_mm.js`?

Comment (Uxio0) [reply docs/04_tying_in_the_market_maker.md]: Out of the blue a block of code appears without a hint of what file it is

Comment (germartinez) [reply docs/04_tying_in_the_market_maker.md]: I mean, I think there is a "/" missing at the beginning :)

Comment (Uxio0) [reply docs/04_tying_in_the_market_maker.md]: Ah ok!

Comment (Uxio0) [reply docs/04_tying_in_the_market_maker.md]: Done

**Pull request merged**